jdaviz/configs/mosviz/helper.py

```diff
@@ -234,14 +234,25 @@ def _zoom_to_slit_params(self, msg):
 
         return pix, pixel_height
 
-    def load_data(self, spectra_1d=None, spectra_2d=None, images=None,
+    def load_data(self, directory=None, instrument=None,
```

Comment:
I think this change needs to be reverted, unless I have missed some discussions. There was a long conversation between @javerbukh and I on why the keyword order should remain as such, because some people like to pass in keywords as positional arguments and we do not use `*` to enforce keyword-only keywords.

Reply:
My main concern was that the only places I see `directory` and `instrument` are the source code and the example Mosviz notebooks. I thought it made sense to mention them in the docstring, and if they were mentioned, I thought it made sense to include them as arguments.

As for argument order, is the default usage of `load_data()` supposed to be load by file(s) or load by searching a directory? Given that each initially leads down a divergent path, I think not enforcing keywords is difficult -- how do you guess whether the first positional argument is supposed to be a 1D spectrum file or a directory?

Reply:
Original discussions at https://github.com/spacetelescope/jdaviz/pull/766#pullrequestreview-724513652 . It would be less controversial if you add the new stuff at the end of the list (before `*args` though the way `*args` is defined, it doesn't really work but that is another story).

Or we can opt to break the API but you'll have to be careful about existing notebooks out there.

Reply:
You make a good point @ojustino , I'm not sure how many people still use `mosviz.load_data(spec1d, spec2d)`  and `mosviz.load_data(directory, intrument)` could be a better change, but we may want to save that for a separate PR. I would not want that discussion to hold up updating the documentation.